opera/genesisstore/fileshash/filehash_test.go

```diff
@@ -71,10 +70,10 @@ func TestFileHash_ReadWrite(t *testing.T) {
 
 func testFileHash_ReadWrite(t *testing.T, content []byte, expRoot hash.Hash, pieceSize uint64) {
 	require := require.New(t)
-	tmpDirPath, err := ioutil.TempDir("", "filehash*")
+	tmpDirPath, err := os.MkdirTemp("", "filehash*")
```

Comment:
Also here, and in other places, [t.TempDir()](https://pkg.go.dev/testing#B.TempDir) would be the more canonical way to deal with temporary directories in tests.